client/js/models/Relationship.ts

```diff
@@ -0,0 +1,13 @@
+import { Issue } from './issue';
+
+export interface RelationshipType {
+  id: number;
+  name: string;
+  label: string;
+}
+
+export interface Relationship {
+  id: number;
+  type: RelationshipType;
+  issue: Issue;
```

Comment:
@waltergar The relationship should use the proper `Issue`, but it can use `IssueRelationshipRef` or equivalent.  This way the proper `Issue` model could enforce the right semantics, e.g. fields like `description` and `project` are mandatory.  It also is strange to have an issue that contains a relationship that contains an issue.